ExampleMod/Content/Items/Weapons/ExampleSpear.cs

```diff
@@ -0,0 +1,48 @@
+﻿using ExampleMod.Content.Projectiles;
+using Terraria;
+using Terraria.ID;
+using Terraria.ModLoader;
+using static Terraria.ModLoader.ModContent;
+
+namespace ExampleMod.Content.Items.Weapons
+{
+	public class ExampleSpear : ModItem
+	{
+		public override void SetStaticDefaults() {
+			Tooltip.SetDefault("An example spear");
+		}
+
+		public override void SetDefaults() { 
+			item.damage = 40; // The damage inflicted by the item.
+			item.useStyle = ItemUseStyleID.Shoot; // The use style of your item.
```

Comment:
Comments should say what the code _doesn't_. Not every line needs a comment if it doesn't add additional information